pycbc/waveform/generator.py

```diff
@@ -44,7 +48,11 @@
 
 
 #
-#   Generator for CBC waveforms
+# =============================================================================
+#
+#                        Radiation-frame generators
+#
+# =============================================================================
```

Comment:
I think this comment should go after the BaseGenerator .... And the Ylm modes are defined in a stationary source frame, not in the radiation frame (e.g. you don't need the orientation of the source w.r.t the observe to define them).

Reply:
Good point. But the modes generators also inherit from `BaseGenerator`, and are in this section. I can't think of a single name for these. Saying source frame isn't really right, since the masses the underlying function use are detector frame. Really, this is the frame where you don't apply a sky location, time shift, or polarization. I couldn't think of a single name for that, so... I just removed the header.